docs/configuration/registration/msgraph.rst

```diff
@@ -47,18 +47,45 @@ Configuration
 =============
 
 1. Obtain the parameters from the Azure application.
-2. In Open Forms, navigate to: **Configuration** > **Overview**. In the **Registration plugin** group, click on **Configuration** for the **Microsoft Graph (OneDrive/SharePoint)** line.
+2. In Open Forms, navigate to: **Configuration** > **Overview**.
 
-   a. Click **Microsoft Graph Service toevoegen**.
-   b. Fill out the form with the requested parameters; note the *Label* field is for internal use.
-   c. Click **Opslaan**
+   a. In the **Registration plugin** section, click on **Configuration** for 
+      the **Microsoft Graph (OneDrive/SharePoint)** line.
+   b. Click the **+**-sign (Add Microsoft Graph Service). A popup should show 
+      up.
+   c. Fill out the form with the requested parameters; note the **Label** field 
+      is for internal use.
+   d. Click **Save** to save this service configuration
+   e. Click **Save** again to select the just added service configuration as 
+      the default service.
 
-3. Navigate to: **Configuration** > **Microsoft Graph registration**
+3. Open any form, and open the **Registration** tab
 
-   a. Select the *Microsoft Graph Service* we created above.
-   b. Click **Opslaan**
+   a. Select **Microsoft Graph (OneDrive/Sharepoint)** as registration backend.
+   b. Optionally fill in the desired **Path** and **Drive ID**. The path value 
+      supports the variables ``{{year}}``, ``{{month}}`` and ``{{day}}`` to 
+      make dynamic paths.
 
-The Microsoft Graph configuration is now completed and can be selected as registration backend in the form builder.
+Submissions on the configured form should now create a folder with the form 
+name, in the configured drive and path. In this folder, the submission PDF, 
```

Comment:
There is also a folder with the submission reference that is created inside the folder with the form name (and then all data/attachments are in there).

Reply:
@SilviaAmAm can you commit the path template in the docs?